frontend/simple/js/router.js

```diff
@@ -2,6 +2,7 @@ import Router from 'vue-router'
 import store from './state'
 import SignUp from '../views/SignUp.vue'
 import CreateGroup from '../views/CreateGroup.vue'
+import GroupDashboard from '../views/group-dashboard.vue'
```

Comment:
Fixed in my branch: https://github.com/dsernst/group-income-simple/commit/3e06f909da78cb131c263af8ac718a053126fdb2